package.json

```diff
@@ -237,6 +237,7 @@
     "comment-json": "4.1.0",
     "fs-extra": "9.1.0",
     "@vscode-logging/logger": "1.2.2",
+    "@sap_oss/task_contrib_types": "3.0.6",
```

Comment:
I think it can be devDependency as it is only types

Reply:
Done